test.sh

```diff
@@ -125,12 +210,9 @@ function runTests() {
     -e CONJUR_AUTHN_API_KEY="$api_key_admin" \
     test \
       bash -c "mvn test"
```

Comment:
Ditto here about if `bash -c` is needed here.

Reply:
Take note here also that we are doing `docker-compose run` here which recreates the test container. Is that wanted here?

Reply:
🥇